test-cypress/integration/repository/ontop-repository.spec.js

```diff
@@ -152,6 +239,131 @@ describe('Ontop repositories', () => {
 
         // Then I expect url to be calculated properly
         OntopRepositorySteps.getUrlInput().should('have.value', 'jdbc:snowflake://someHostName.snowflakecomputing.com:1234/?warehouse=test_database');
-
     });
+
+    function assertRepositoryStatus(repositoryId, status) {
```

Comment:
Why is this here and not in some steps or command file? Did you check if there is no such validation already?